recipe/meta.yaml

```diff
@@ -32,6 +32,7 @@ requirements:
     - structlog
     - pytorch >=1.5.0
     - tqdm
+    - transformers
```

Comment:
@hadim, this seems to be optional
https://github.com/ElementAI/baal/blob/26bde3a9ef68454cb2795de3784cde4032fc0391/src/baal/transformers_trainer_wrapper.py

Reply:
True but the import fails during CI. Because of the module is imported at https://github.com/ElementAI/baal/blob/26bde3a9ef68454cb2795de3784cde4032fc0391/src/baal/__init__.py without a conditional check for `transformers`.

I will just remove the import test in the recipe. It's better than adding `transformers` as a dep.

Reply:
Thinking twice about it, it's not a good solution since the package just can't be used without `transformers`.

Reply:
Will report upstream and hold here.

Reply:
Upstream ticket https://github.com/ElementAI/baal/issues/114